docs/hoon/twig/wut-test/bar-or.md

```diff
@@ -10,13 +10,25 @@ sort: 9
 ## Expands to
 
 ```
+:loop
 :ifno  p  
```

Comment:
Okay yeah they're not, having the `$` without a referent is definitely
worse. Thank you for stumbling through our docs trying to bring light to
the gaps in them :3

On Thursday, 23 June 2016, Joe Bryan <notifications@github.com> wrote:

> In docs/hoon/twig/wut-test/bar-or.md
> <https://github.com/urbit/docs/pull/55#discussion_r68286244>:
>
> > @@ -10,13 +10,25 @@ sort: 9
> >  ## Expands to
> >
> >  ```
> > +:loop
> >  :ifno  p
>
> These changes adding |- to the ? runes are inconsistent, but I don't
> think they're worse than what was there before. I puzzled over those
> expansions for a while, trying to figure out if $ could actually recurse
> into a conditional ...
>
> But maybe that's just me ;)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/urbit/docs/pull/55/files/8c309518d1325589ec5ca032103f436687191997#r68286244>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABxXhrpALPH-Cu0saYqgAqpUjOpGUV97ks5qOszOgaJpZM4I9D6c>
> .
>